public/js/pimcore/elementservice.js

```diff
@@ -482,7 +482,7 @@ pimcore.elementservice.editObjectKeyComplete = function (options, button, value,
                 var index, record;
                 for (index = 0; index < affectedNodes.length; index++) {
                     record = affectedNodes[index];
-                    pimcore.elementservice.refreshNode(record);
+                    pimcore.elementservice.refreshNode(record.parentNode);
```

Comment:
if the `parentNode` refreshing approach is valid, need to be refactored to gather all the parentNodes in one array and refresh these, instead of refreshing for each affected node, since these nodes could share the same parents, no need to refresh them multiple times.